packages/abcm2ps/PKGBUILD

```diff
@@ -0,0 +1,33 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: Christopher Arndt <aur -at- chrisarndt -dot- de>
+# Contributor: Tuyen Pham <tuyenpm9@gmail.com>
+# Contributor: Lex Black <autumn-wind@web.de>
+# Contributor: Simon Thorpe <simon@hivetechnology.com.au>
+# Contributor: kess <joe@macmahon.eu>
+# Contributor: damir <damir@archlinux.org>
+# Contributor: Luis Pablo Gasparotto <lpgasparotto(at)gmail(dot)com>
+
+pkgname=abcm2ps
+pkgver=8.14.13
+pkgrel=2
+pkgdesc='Convert ABC music notation files to PostScript from the commandline'
+arch=(x86_64 aarch64)
+url='http://moinejf.free.fr/'
+license=(GPL3)
+depends=(pango)
+makedepends=(python-docutils)
+conflicts=(abcm2ps-devel)
```

Comment:
This is also taken from the AUR? `PKGBUILD. I'm not sure when a package name `abcm2ps-devel` ever existed. And if it did, it would be the responsibility of that package to declare the conflict to the release package, IMHO. Leave it in or not?

Reply:
abcm2ps-devel sounds like a debian package. Maybe a converted package? 
If abcm2ps-devel does not exist in AUR I would consider it an error, and remove it.

Reply:
It existed at some point: https://github.com/aur-archive/abcm2ps-devel

Reply:
I've taken it out. Seems unlikely somebody still has that package installed.